### ISSUE TITLE
IE11: css can not be accessed

### ISSUE THREAD

**Issue by h0jeZvgoxFepBQ2C** [nanostudio-org/nanogallery2#160]
```
    function ViewerToolsOpacity( op ) {
      G.VOM.$toolbar.css('opacity', op);
      if( G.VOM.$toolbarTL != null ) {
        G.VOM.$toolbarTL.css('opacity', op);
      }
      if( G.VOM.$toolbarTR != null ) { 
        G.VOM.$toolbarTR.css('opacity', op);
      }
      G.VOM.$content.find('.nGY2ViewerAreaNext').css('opacity', op);
      G.VOM.$content.find('.nGY2ViewerAreaPrevious').css('opacity', op);
    }
```

should be

 ```
 function ViewerToolsOpacity( op ) {
      if( G.VOM.$toolbar != null ) {
        G.VOM.$toolbar.css('opacity', op);
      }
      if( G.VOM.$toolbarTL != null ) {
        G.VOM.$toolbarTL.css('opacity', op);
      }
      if( G.VOM.$toolbarTR != null ) { 
        G.VOM.$toolbarTR.css('opacity', op);
      }
      G.VOM.$content.find('.nGY2ViewerAreaNext').css('opacity', op);
      G.VOM.$content.find('.nGY2ViewerAreaPrevious').css('opacity', op);
    }
```

**Comment by Kris-B** [comment]
fixed in v2.4.0 - thx!

**Comment by Kris-B** [comment]
Please note that package for v2.4.0 was corrupted - please use v2.4.1
